release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1363,6 +1363,58 @@ This script removes unauthenticated subjects from the following cluster role bin
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2060492[*BZ#2060492*])
+
+* In RAN vDU deployments, PTP is used instead of NTP. In order to use PTP in the cluster, you must disable NTP. Currently, this is done using an additional `MachineConfig` CR to disable `chronyd` in the cluster.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2058231[*BZ#2058231*])
+
+* In a node with a dual NIC PTP configuration, the PTP events framework reports an incorrect status of `HANDOVER` or `FREERUN` instead of a `LOCKED` state for a PTP clock in a `LOCKED` state. As a result, consumer applications using PTP events get an incorrect status for the PTP clock.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2092275[*BZ#2092275*])
+
+* In a node with a single NIC boundary clock PTP configuration, the PTP events framework reports an incorrect status of `FREERUN` instead of a `LOCKED` state for a PTP clock in a `LOCKED` state. As a result, consumer applications using PTP events get an incorrect status for the PTP clock.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2091441[*BZ#2091441*])
+
+* When you reset a network interface that is configured as a PTP ordinary clock, only the PTP `class-change` event is recorded in the PTP events log. `status-change` or `sync-status` events are not recorded in the PTP events log. The failure of the PTP events framework to capture these events does not impact PTP network timekeeping.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2091437[*BZ#2091437*])
+
+* When you apply a `PTPConfig` CR in the cluster that configures two slave interfaces, PTP metrics and events list the network interfaces with a `SLAVE` and `MASTER` status instead of `SLAVE` and `PASSIVE` respectively. To diagnose problems with network interfaces configured with PTP, use the PTP Management Client (`pmc`) tool.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2091434[*BZ#2091434*])
+
+* In a cluster configured for PTP, `phc2sys` reports offset values for master ports. PTP events are generated for the offsets. For example:
++
+[source,terminal]
+----
+phc2sys[859300.541]: [ptp4l.0.config] ens3f1 rms 52 max 52 freq 3081 +/ 0 delay 2542 +/- 0
+----
++
+Offset metrics and events should not be reported for master ports.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2047308[*BZ#2047308*])
+
+* In {product-title} 4.11, PTP events publisher and subscriber resource addresses are not implemented as per O-RAN specifications.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2037991[*BZ#2037991*])
+
+* When the Special Resource Operator (SRO) is deployed with Operator Lifecycle Manager (OLM) using the {product-title} v4.11 brew bundle image, the `SpecialResource` CRD is installed but the `SpecialResourceModule` and `PreflightValidation` CRDs are missing. The SRO pod continuously restarts.
```

Comment:
@ybettan @yliu127 Plz review. Comment below.

Reply:
Please remove this one. It's already fixed.

Reply:
Removed.

Reply:
done